sass/components/_theme_variables.css

```diff
@@ -0,0 +1,41 @@
+:root {
+    --surface-color: #ffffff;
+    --background-color: #ffffff;
+
+    --font-color-main: rgba(0, 0, 0, 0.87);
+    --font-color-medium: rgba(0, 0, 0, 0.56);
+    --font-color-disabled: rgba(0, 0, 0, 0.38);
+
+    --hover-color: rgba(0, 0, 0, 0.04);
+    --focus-color: rgba(0, 0, 0, 0.12);
+    --focus-color-solid: #E0E0E0;
+
+    --background-color-disabled: rgba(0, 0, 0, 0.12);
+    --background-color-level-4dp: rgba(0, 0, 0, 0.09);
+
+    --separator-color: #DDDDDD; /* borders between components */
+
+    --slider-track-color: rgba(0, 0, 0, 0.32);
+    --switch-thumb-off-color: #ffffff;
+}
+
```

Comment:
does the user have to define this or are this the standard colors?

Reply:
These will be standard colors for light and dark theme.
A user would be able to override those or create his own scheme, by adding a css file with `:root[theme='my_theme'] { ... }`
and using `document.documentElement.setAttribute('theme', 'my_theme');`